### ISSUE TITLE
dpkg error on Pop!_OS 22.04 LTS - GNOME 42.1

### ISSUE THREAD

**Issue by RationalG** [yilozt/mutter-rounded#64]
Hello,
When I try to run dpkg after running the package.sh script, I get an error message warning me that it could break libgtk-3-0.

This is the command I run : `sudo dpkg -i libmutter-10-0*.deb mutter-common*.deb`

And here is the output : 

```
dpkg: regarding libmutter-10-0_42.1-0ubuntu1_amd64.deb containing libmutter-10-0:amd64:
 libmutter-10-0:amd64 breaks libgtk-3-0 (<< 3.24.33-1ubuntu2~)
  libgtk-3-0:amd64 (version 3.24.33-1ubuntu1) is present and installed.
dpkg: error processing archive libmutter-10-0_42.1-0ubuntu1_amd64.deb (--install):
 **installing libmutter-10-0:amd64 would break libgtk-3-0:amd64, and
 deconfiguration is not permitted (--auto-deconfigure might help)**
(Reading database ... 347110 files and directories currently installed.)
Preparing to unpack mutter-common_42.1-0ubuntu1_all.deb ...
Unpacking mutter-common (42.1-0ubuntu1) over (42.1-0ubuntu1) ...
Setting up mutter-common (42.1-0ubuntu1) ...
Processing triggers for libglib2.0-0:i386 (2.72.1-1) ...
Processing triggers for libglib2.0-0:amd64 (2.72.1-1) ...
Processing triggers for man-db (2.10.2-1) ...
Errors were encountered while processing:
 libmutter-10-0_42.1-0ubuntu1_amd64.deb

```
I also want to precise that when I am running package.sh I encounter a gpp error when it is running debsign, but it doesn't seem to be a showstopper : 

```
W: mutter source: format-3.0-but-debian-changes-patch
W: libmutter-10-0: symbols-file-contains-debian-revision on symbol clutter_frame_get_hints@Base and 4 others (libmutter-clutter-10.so.0)
W: libmutter-10-0: symbols-file-contains-debian-revision on symbol cogl_onscreen_count_pending_frames@Base (libmutter-cogl-10.so.0)
W: libmutter-10-0: symbols-file-contains-debian-revision on symbol meta_prefs_get_blur_brightness@Base and 11 others (libmutter-10.so.0)
N: 3 hints overridden (3 info); 0 unused overrides
Finished running lintian.
**Now signing changes and any dsc files...
 signfile dsc mutter_42.1-0ubuntu1.dsc Jeremy Bicha <jbicha@ubuntu.com>
gpg: skipped "Jeremy Bicha <jbicha@ubuntu.com>": No secret key
gpg: /tmp/debsign.yf2EmFLm/mutter_42.1-0ubuntu1.dsc: clear-sign failed: No secret key
debsign: gpg error occurred!  Aborting....
debuild: fatal error at line 1112:
running debsign failed**
[run] cd ../
[run] mv gir1.2-mutter-10_42.1-0ubuntu1_amd64.deb libmutter-10-0_42.1-0ubuntu1_amd64.deb libmutter-10-dev_42.1-0ubuntu1_amd64.deb libmutter-test-10_42.1-0ubuntu1_amd64.deb mutter-10-tests_42.1-0ubuntu1_amd64.deb mutter_42.1-0ubuntu1_amd64.deb mutter-common_42.1-0ubuntu1_all.deb ../
build finish, you can find packages at: /home/dwarfpriest/Documents/Repos/mutter-rounded/ubuntu
```

I am not sure if it could break my system, but if that's the case I will make a backup that's for sure.

**Comment by yilozt** [comment]
Fix via commit [c8604ea](https://github.com/yilozt/mutter-rounded/commit/c8604ea18b3d0860c901fd81151d10d42133daaf).

You can download deb packages from [Github Action](https://github.com/yilozt/mutter-rounded/actions/runs/2366548004) if you don't want to rebuild by yourself.

**Comment by RationalG** [comment]
Thank you ! I am going to try and will come back to you in a short time

**Comment by RationalG** [comment]
Thank you yilozt !!! Everything seem to work perfectly. Just a minor problem, is that now whenever I want to install a package I am prompted with the following message : 

```
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 gir1.2-mutter-10 : Depends: libmutter-10-0 (= 42.0-3ubuntu2pop0~1650489643~22.04~ed156a0) but 42.0-3ubuntu2 is to be installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

I am not sure about the risk taken if I run this command. But I guess I can live with that.

**Comment by yilozt** [comment]
I have update build script in [923fe2a](https://github.com/yilozt/mutter-rounded/commit/923fe2a3389265924f3f11a5f9b6d5c4a69f23e2), it should be built and installed properly in Pop!_OS 22.04 now.

Deb packages from Ubuntu 22.04 can be installed Pop!_OS properly. I have not found docker image of Pop!_OS, so I can't build it with Github Actions.

You can run `sudo apt install libmutter-10-0 mutter-common` to restore offical mutter first, then use `git pull` to update the build script, then run `package.sh` to rebuild by yourself.

**Comment by RationalG** [comment]
You're the man yilozt. Everything works fine with your last commit. Good job dude, this issue is resolved now :)

By the way, just FYI, I didn't have to restore official mutter first. It's weird, but after a reboot, the system was actually back with the official mutter package restored. I lost all the blur effects & the rounded corners, and inside dconf, only the blur keys were remaining but they were not bound to a schema anymore so they weren't effective. I think the dependencies auto-fixed themselves when I ran apt-get update some hours ago but I am not sure.

When I rebuilded and installed with your last commit, all my previous settings were back so it's cool ;)
